scripts/vagrant/provision/manifests/kube-prometheus/grafana-dashboardDefinitions.yaml

```diff
@@ -943,6 +943,6534 @@ items:
   metadata:
     name: grafana-dashboard-k8s-cluster-rsrc-use
     namespace: monitoring
+- apiVersion: v1
+  data:
+    m3db_node_details.json: |-
+      {
```

Comment:
Is there a way we can pull this out into a separate json representation of the dash and include that here instead?

Reply:
I don't think so, I couldn't find anything. It looks like this is the way to do it if we write out the yamls ourselves. 

Although it seems like using `kubectl` you can create config maps from files with a flag.

Reply:
Maybe even just as part of a build step that will swap out a template var for the dash json or something?

Reply:
We could prob use perl or sed but I think it's too much for these test dash defs. 

All other dash are also defined in the same way in the file.

Reply:
kk